onnxruntime/test/python/onnxruntime_test_python_iobinding.py

```diff
@@ -64,19 +45,19 @@ def test_bind_input_only(self):
         ort_output = io_binding.copy_outputs_to_cpu()[0]
 
         # Validate results
-        self.assertTrue(np.array_equal(torch_output_vals, ort_output))
+        self.assertTrue(np.array_equal(self.create_expected_output(), ort_output))
 
     def test_bind_input_to_cpu_arr(self):
-        torch_input, torch_output, torch_output_vals = self.create_model_and_input()
+        input = self.create_numpy_input()
 
-        session = onnxruntime.InferenceSession('model.onnx')
+        session = onnxruntime.InferenceSession(get_name("mul_1.onnx"))
         io_binding = session.io_binding()
         
         # Bind Numpy object (input) to CUDA
-        io_binding.bind_cpu_input('input', torch_input.cpu().detach().numpy())
+        io_binding.bind_cpu_input('X', self.create_numpy_input())
```

Comment:
nit: is it binding to CUDA or CPU?

Reply:
it is binding input ON cpu but the underlying ortvalue that is created maybe bound to cuda if the consumer nodes are all on cuda isn't it ?

Reply:
Maybe the comment should say that it's binding an input that's on cpu to wherever the model needs it (may or may not be CUDA depending on where the node/s that need it were partitioned to).

Reply:
sure - I think this is cleanup from the old PR

Reply:
Fixed.